### PR TITLE
Design fixes

### PR DESCRIPTION
**Pull request by LouaiKhodary13**

### CODE REVIEW

File: src/Components/Forms/BillboardUploadAd.tsx [hunk 1]
```diff
@@ -100,48 +109,52 @@ const BillboardUploadAd = () => {
...
-      setUploadScriptForm((prevState) => ({
-        ...prevState,
-        contributors: [...prevState.contributors, newContributor],
-      }));
-    }
-  };
-  const handlePdfFileChange = (e) => {
+  // const handleAddContributors = () => {
+  //   const newContributor = prompt('Enter the name of the contributor');
+  //   if (newContributor) {
```
Comment (Risidio) [inline src/Components/Forms/BillboardUploadAd.tsx:133]: can you remove unused code, so we can keep the code clean

File: src/Components/Forms/BillboardUploadAd.tsx [hunk 2]
```diff
@@ -234,7 +242,7 @@ const BillboardUploadAd = () => {
     user.userName.toLowerCase().includes(searchInputUsers.toLowerCase())
   );
 
-  const handleAddUser = (userName: string, role: string, status: string) => {
+  const handleAddUser = (userName: string, role: string[], status: string) => {
```
Comment (Risidio) [inline src/Components/Forms/BillboardUploadAd.tsx:224]: use an interface to declare the props and pass it in

File: src/Components/Forms/BillboardUploadAd.tsx [hunk 3]
```diff
@@ -244,15 +252,15 @@ const BillboardUploadAd = () => {
       // Add a new collaborator with the provided userName, role, and status
       const newUser = {
         userName,
-        role, // You can provide a default role for the new user
-        status, // You can provide a default status for the new user
+        roles: role, // Assuming roles is the correct property name for the role array
+        status,
       };
-
+      console.log(newUser);
```
Comment (Risidio) [inline src/Components/Forms/BillboardUploadAd.tsx:238]: remove console.log

File: src/Components/Forms/BillboardUploadAd.tsx [hunk 4]
```diff
@@ -244,15 +252,15 @@ const BillboardUploadAd = () => {
...
       // Update the collaboratorsArray using the state setter
-      setFetchedCollaborators((prevCollaborators) => [
-        ...prevCollaborators,
-        newUser,
-      ]);
+      // setFetchedCollaborators((prevCollaborators) => [
```
Comment (Risidio) [inline src/Components/Forms/BillboardUploadAd.tsx]: remove unused code

File: src/Layouts/Popup/BillboardPopup/Billboard.tsx [hunk 5]
```diff
@@ -200,6 +199,9 @@ const Billboard: React.FC = () => {
                     ></div>
                   ))}
                 </div>
+                {/* <h2 className='mt-2 font-jost font-[300] text-[17px] text-[#FFFFFF]'>
+                  The citizens who've done the most for PROM
```
Comment (Risidio) [inline src/Layouts/Popup/BillboardPopup/Billboard.tsx:203]: remove unused code

File: src/Layouts/Popup/Call/ActiveCall.tsx [hunk 6]
```diff
@@ -1,72 +1,90 @@
...
-            </p>
-
-            
+    <div className={classNames('call call--active', {})}>
+      <div className='call__left'>
+        <div className='call-img'></div>
+        <div className='call__text'>
+          <h1 className='call__text-title'>{userName}</h1>
+          <div className='call__text-duration'>
+            <svg
```
Comment (Risidio) [inline src/Layouts/Popup/Call/ActiveCall.tsx:20]: import svg as images ,   remove SVG code from file

File: src/Layouts/Popup/Call/ActiveCall.tsx [hunk 7]
```diff
@@ -1,72 +1,90 @@
...
-            <rect x="0.5" y="1.18896" width="51.1158" height="49.7167" rx="24.8583" stroke="#B6BAC4" />
+      <div className='call__buttons'>
+        <button type='button' className='call__button--incoming'>
+          <svg
```
Comment (Risidio) [inline src/Layouts/Popup/Call/ActiveCall.tsx:39]: import as an image

File: src/Layouts/Popup/NavBarPopup/BagPopup.tsx [hunk 8]
```diff
@@ -65,8 +65,8 @@ const BagPopup = () => {
                 />
                 <path
                   id='Vector_2'
-                  fill-rule='evenodd'
-                  clip-rule='evenodd'
+                  fillRule='evenodd'
```
Comment (Risidio) [inline src/Layouts/Popup/NavBarPopup/BagPopup.tsx:68]: import as an image

File: src/Layouts/Popup/NavBarPopup/FriendsPopup.tsx [hunk 9]
```diff
@@ -26,83 +25,80 @@ const FriendsPopup: React.FC<Props> = ({
...
+          <i className='contact__form-search-icon'>
+            <svg
+              xmlns='http://www.w3.org/2000/svg'
```
Comment (Risidio) [inline src/Layouts/Popup/NavBarPopup/FriendsPopup.tsx:54]: import as an image instaead

File: src/Components/Forms/BillboardUploadAd.tsx (hunk 1 above)
Comment (LouaiKhodary13) [reply src/Components/Forms/BillboardUploadAd.tsx]: done

File: src/Components/Forms/BillboardUploadAd.tsx [hunk 10]
```diff
@@ -199,11 +212,6 @@ const BillboardUploadAd = () => {
       formData.append(`image_${index}`, image);
     });
 
-    //  cannot directly log FormData objects, loop through the entries
-    for (var pair of formData.entries()) {
-      console.log(pair[0] + ', ' + pair[1]);
-    }
-
     // Submit the form data
     console.log('Submitting form data...');
```
[the remark this answers is not in the document]
Comment (LouaiKhodary13) [reply src/Components/Forms/BillboardUploadAd.tsx]: removed

File: src/Layouts/Popup/NavBarPopup/MessagePopup.tsx [hunk 11]
```diff
@@ -1,19 +1,56 @@
+import { useState } from 'react';
 import avatar from '../../../assets/avatarSmall.svg';
+// import EmojiPicker from 'emoji-picker-react';
 // using default component caused problems such as duplication errors this why I made my own style for the triangle
```
Comment (Risidio) [inline src/Layouts/Popup/NavBarPopup/MessagePopup.tsx:4]: remove unused code

File: src/Layouts/Popup/NavBarPopup/MessagePopup.tsx [hunk 12]
```diff
@@ -1,19 +1,56 @@
...
+  // const [message, setMessage] = useState<string>('');
+  // const [sendMessages, setSendMessages] = useState([]);
+  // emojis for the chats
+
+  // handling the emoji
+  // const handleShowEmoji = () => {
+  //   setShowEmojiPicker(!showEmojiPicker);
+  // };
+  // const handleEmojiClick = (emojiObject: EmojiClickData) => {
+  //   const emoji = emojiObject.emoji;
```
Comment (Risidio) [inline src/Layouts/Popup/NavBarPopup/MessagePopup.tsx:33]: remove unused code

File: src/Layouts/Popup/NavBarPopup/MessagePopup.tsx [hunk 13]
```diff
@@ -247,6 +292,23 @@ const MessagePopup = () => {
...
+        style={{
+          position: 'absolute',
+          zIndex: '100',
+          left: '50%',
+          top: '50%',
+        }}
+      >
+        {showEmojiPicker && (
+          <EmojiPicker
+            className='chat__emoji'
```
Comment (Risidio) [inline src/Layouts/Popup/NavBarPopup/MessagePopup.tsx:305]: please remove unused code

File: src/Layouts/Popup/NavBarPopup/ProfilePopup.tsx [hunk 14]
```diff
@@ -12,141 +12,175 @@ import UserProfile from './UserProfile';
...
+            xmlns='http://www.w3.org/2000/svg'
+          >
+            <path
+              id='Vector'
+              d='M46 8H6C5.46957 8 4.96086 7.78929 4.58579 7.41421C4.21071 7.03914 4 6.53043 4 6C4 5.46957 4.21071 4.96086 4.58579 4.58579C4.96086 4.21071 5.46957 4 6 4H40C40.5304 4 41.0391 3.78929 41.4142 3.41421C41.7893 3.03914 42 2.53043 42 2C42 1.46957 41.7893 0.96086 41.4142 0.585787C41.0391 0.210714 40.5304 0 40 0H6C4.4087 0 2.88258 0.632141 1.75736 1.75736C0.632141 2.88258 0 4.4087 0 6V38C0 39.5913 0.632141 41.1174 1.75736 42.2426C2.88258 43.3679 4.4087 44 6 44H46C47.0609 44 48.0783 43.5786 48.8284 42.8284C49.5786 42.0783 50 41.0609 50 40V12C50 10.9391 49.5786 9.92172 48.8284 9.17157C48.0783 8.42143 47.0609 8 46 8ZM37 28C36.4067 28 35.8266 27.8241 35.3333 27.4944C34.8399 27.1648 34.4554 26.6962 34.2284 26.148C34.0013 25.5999 33.9419 24.9967 34.0576 24.4147C34.1734 23.8328 34.4591 23.2982 34.8787 22.8787C35.2982 22.4591 35.8328 22.1734 36.4147 22.0576C36.9967 21.9419 37.5999 22.0013 38.148 22.2284C38.6962 22.4554 39.1648 22.8399 39.4944 23.3333C39.8241 23.8266 40 24.4067 40 25C40 25.7956 39.6839 26.5587 39.1213 27.1213C38.5587 27.6839 37.7957 28 37 28Z'
```
Comment (Risidio) [inline src/Layouts/Popup/NavBarPopup/ProfilePopup.tsx:155]: import as image

Comment (LouaiKhodary13) [reply src/Layouts/Popup/NavBarPopup/ProfilePopup.tsx]: done

File: src/Layouts/Popup/NavBarPopup/MessagePopup.tsx (hunk 13 above)
Comment (LouaiKhodary13) [reply src/Layouts/Popup/NavBarPopup/MessagePopup.tsx]: removed

File: src/Layouts/Popup/NavBarPopup/MessagePopup.tsx (hunk 12 above)
Comment (LouaiKhodary13) [reply src/Layouts/Popup/NavBarPopup/MessagePopup.tsx]: removed

File: src/Layouts/Popup/NavBarPopup/MessagePopup.tsx (hunk 11 above)
Comment (LouaiKhodary13) [reply src/Layouts/Popup/NavBarPopup/MessagePopup.tsx]: removed

File: src/Layouts/Popup/NavBarPopup/FriendsPopup.tsx (hunk 9 above)
Comment (LouaiKhodary13) [reply src/Layouts/Popup/NavBarPopup/FriendsPopup.tsx]: done

File: src/Layouts/Popup/NavBarPopup/BagPopup.tsx (hunk 8 above)
Comment (LouaiKhodary13) [reply src/Layouts/Popup/NavBarPopup/BagPopup.tsx]: done

File: src/Layouts/Popup/Call/ActiveCall.tsx (hunk 7 above)
Comment (LouaiKhodary13) [reply src/Layouts/Popup/Call/ActiveCall.tsx]: done

File: src/Layouts/Popup/Call/ActiveCall.tsx (hunk 6 above)
Comment (LouaiKhodary13) [reply src/Layouts/Popup/Call/ActiveCall.tsx]: done

File: src/Layouts/Popup/BillboardPopup/Billboard.tsx (hunk 5 above)
Comment (LouaiKhodary13) [reply src/Layouts/Popup/BillboardPopup/Billboard.tsx]: removed

File: src/Components/Forms/BillboardUploadAd.tsx (hunk 4 above)
Comment (LouaiKhodary13) [reply src/Components/Forms/BillboardUploadAd.tsx]: removed

File: src/Components/Forms/BillboardUploadAd.tsx (hunk 3 above)
Comment (LouaiKhodary13) [reply src/Components/Forms/BillboardUploadAd.tsx]: removed

File: src/Components/Forms/BillboardUploadAd.tsx (hunk 2 above)
Comment (LouaiKhodary13) [reply src/Components/Forms/BillboardUploadAd.tsx]: done

**Pull request merged**